src/main/java/aspects/AuthAspect.java

```diff
@@ -0,0 +1,30 @@
+package aspects;
+
+import beans.AuthenticatedRequestBean;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.springframework.beans.factory.annotation.Autowired;
+import services.AuthService;
+
+import java.util.Arrays;
+
+/**
+ * Aspect to configure the AuthService
+ */
+@Aspect
+public class AuthAspect {
+
+    @Autowired
+    protected AuthService authService;
+
+    @Before(value = "@annotation(org.springframework.web.bind.annotation.RequestMapping)")
```

Comment:
Currently it will attempt to assign authentication for every request, but won't fail if its not an AuthenticatedRequest. It would fail later when the expected `Auth` is null